pkg/cmd/roachtest/tests/online_restore.go

```diff
@@ -567,6 +567,13 @@ func runRestore(
 			return errors.Wrapf(err, "failed to add some empty tables")
 		}
 		restoreStartTime = timeutil.Now()
+		if runOnline && sp.linkPhaseTimeout != 0 {
+			cancel := roachtestutil.GoAfter(ctx, sp.linkPhaseTimeout, func() {
```

Comment:
Isn't isomorphic to `time.AfterFunc`?

Reply:
It is! I replaced this with time.AfterFunc.